pkg/router/routeapihelpers/validation_test.go

```diff
@@ -786,6 +786,28 @@ phDhW4aOnrygBwYFK4EEACKhZANiAARQpQW0gA2TGexsYDtm9APVNFzWGZd2+/yX
 kRsRwP+6Lr8hPeipqnjaxQGn3Je+We2haNgApCNU3AaSye1epeqAUaMREONr3F8Y
 h7Z/ugyUzsyUY8JR3CNlk16SnBmVW0o=
 -----END EC PRIVATE KEY-----
+`
```

Comment:
nit: if not too much trouble, could you add godoc for generating a MD5 cert for QE or future maintainers?

Reply:
Yea good point, that might be helpful. I also isolated the key to keep things clean.